src/main/resources/db/migration/V1_57__update_audit_log_data_field_type.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE audit_log
```

Comment:
Downtime during release

Reply:
How can we know how long?

Reply:
During the deployment from the time the new node starts up the application until the old node is drained of all connections.